lib/reek/configuration/app_configuration.rb

```diff
@@ -79,6 +83,10 @@ def load_values(configuration_hash)
         end
       end
 
+      def initialize(configuration_hash: nil)
```

Comment:
I'd consider encoding the data type "hash" into the variable name an antipattern in Ruby.

Reply:
Fixed. Ready for another round.